server/middlewares/stats.js

```diff
@@ -71,13 +110,27 @@ const trackTxSubmissions = mung.json((body, req, res) => {
         : 'otherTxSubmissions'
     const txSubmissionSuccess = body.Right ? 'successful' : 'unsuccessful'
 
-    incrCountersBy(`${txSubmissionType}:${txSubmissionSuccess}`, 1)
-
-    if (txSubmissionSuccess === 'successful') {
-      const {txBody} = req.body
-      const txOutAmount = parseTxBodyOutAmount(txBody)
-
-      incrCountersBy(`${txSubmissionType}:sentOut`, txOutAmount)
+    try {
+      if (txSubmissionSuccess === 'successful') {
+        const {txBody} = req.body
+        const txOutAmount = parseTxBodyOutAmount(txBody)
+
+        trackEvent({
+          eventCategory: 'Payments',
+          eventAction: `${txSubmissionType}:sentOutSuccess`,
+          eventLabel: 'successful payment',
+          eventValue: txOutAmount,
+        })
+      } else {
+        trackEvent({
+          eventCategory: 'Payments',
```

Comment:
prefixing the key names with "event" seems redundant, if the function is called `trackEvent`